backend/wmg/data/snapshot.py

```diff
@@ -106,7 +110,11 @@ def _load_primary_filter_data(snapshot_identifier: str) -> Dict:
 
 
 def _load_dataset_to_gene_ids_data(snapshot_identifier: str) -> Dict:
-    return json.loads(_read_s3obj(f"{snapshot_identifier}/{DATASET_TO_GENE_IDS_FILENAME}"))
+    try:
+        return json.loads(_read_s3obj(f"{snapshot_identifier}/{DATASET_TO_GENE_IDS_FILENAME}"))
+    except:
```

Comment:
Anyone know what error to catch here?

Reply:
I believe its botocore.exceptions.ClientError

Reply:
assuming this is meant to return a 404 from aws